client/components/jetpack/connection-health/index.tsx

```diff
@@ -17,23 +23,35 @@ import {
 	XMLRPC_ERROR,
 } from './constants';
 import { ErrorNotice } from './error-notice';
-import { useCheckJetpackConnectionHealth } from './use-check-jetpack-connection-health';
 
 interface Props {
 	siteId: number;
 }
 
 export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 	const translate = useTranslate();
+	const dispatch = useDispatch();
 	const siteIsAutomatedTransfer = useSelector(
 		( state ) => !! isSiteAutomatedTransfer( state, siteId )
 	);
 
-	const {
-		data: jetpackConnectionHealth,
-		isLoading: isLoadingJetpackConnectionHealth,
-		isError: isErrorCheckJetpackConnectionHealth,
-	} = useCheckJetpackConnectionHealth( siteId );
+	const isErrorCheckJetpackConnectionHealth = useSelector( ( state ) =>
+		getJetpackConnectionHealthRequestError( state as AppState, siteId )
+	);
+
+	const isLoadingJetpackConnectionHealth = useSelector( ( state ) =>
+		isRequestingJetpackConnectionHealthStatus( state as AppState, siteId )
+	);
+
+	const jetpackConnectionHealth = useSelector( ( state ) =>
+		getJetpackConnectionHealth( state as AppState, siteId )
+	);
+
+	useEffect( () => {
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
How we could remove this ignore?

Reply:
Why do you need the ignore in the first place?

Reply:
It seems that there was a ts error in my IDE, which now is not there. Strange. We don't need this `ts-ignore`, so I'll remove it.